src/api/layers/GeoAdminGeoJsonLayer.class.js

```diff
@@ -44,6 +47,7 @@ export default class GeoAdminGeoJsonLayer extends GeoAdminLayer {
             geoJsonUrl = null,
             styleUrl = null,
             updateDelay = null,
+            maxResolution = DEFAULT_MAX_GEOADMIN_RESOLUTION,
```

Comment:
Same as for KML and GPX is this really needed here do we have a resolution limitation ?